packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -676,10 +697,12 @@ class DefaultMutationMapperStore implements MutationMapperStore {
 
     readonly ptmData: MobxPromise<PostTranslationalModification[]> = remoteData(
         {
-            await: () => [this.mutationData],
+            await: () => [this.mutationData, this.activeTranscript],
             invoke: async () => {
-                if (this.activeTranscript) {
-                    return this.dataFetcher.fetchPtmData(this.activeTranscript);
+                if (this.activeTranscript.result) {
```

Comment:
is it actually possible that there can be no active transcript result (after isComplete)

Reply:
the `activeTranscript` might be undefined, so I have the check here